lectures/newtons_method.md

```diff
@@ -196,33 +196,30 @@ automatic differentiation, and a GPU, we obtain a relatively small error for
 this very large problem in just a few seconds:
 
 ```{code-cell} python3
-# Use tic toc from quantecon to measure runtime
-qe.tic()
+%%time
 p = newton(lambda p: e(p, A, b, c), init_p).block_until_ready()
-time1 = qe.toc()
 ```
 
 ```{code-cell} python3
-np.max(np.abs(e(p, A, b, c)))
+jnp.max(jnp.abs(e(p, A, b, c)))
 ```
 
 With the same tolerance, SciPy's `root` function takes much longer to run,
 even with the Jacobian supplied.
 
 
 ```{code-cell} python3
-qe.tic()
+%% time
```

Comment:
I guess this might be the issue. This should be `%%time`?  Reference: https://ipython.readthedocs.io/en/stable/interactive/magics.html#magic-timeit

Reply:
This gives error on IPython:
```py
In [2]: %% time
   ...: x = pow(2, 3)
   ...: y = pow(4, 5)
   ...: 
   ...: 
UsageError: Cell magic `%%` not found.
```

Reply:
I ran into this problem once and I noticed this error.

But the one I met is similar to [this](https://github.com/ipython/ipython/issues/11659/).

Still many thanks @Smit-create and I've updated your changes.